src/components/Section/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```

Comment:
I think this component should be part of the Card one

Reply:
I don't want to mix them tho. They serve different purposes, like you can have many cards next to each other, but a section is just a ... section 😄 . But I extended the container with `Card` so it can have the same `box-shadow` props tho. Maybe it'll come handy.

Reply:
👌